packages/scaffolder-cli/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# @scaffolder/cli
```

Comment:
Same here, probably replace all usages of `@scaffolder/cli` with `@backstage/scaffolder-cli`